google-cloud-pubsub/samples/acceptance/topics_test.rb

```diff
@@ -35,6 +43,126 @@
 require_relative "../pubsub_resume_publish_with_ordering_keys.rb"
 require_relative "../pubsub_set_topic_policy.rb"
 require_relative "../pubsub_test_topic_permissions.rb"
+require_relative "../pubsub_update_topic_type.rb"
+
+describe "emulator" do
+  let(:pubsub) { Google::Cloud::Pubsub.new }
+  let(:topic_id) { random_topic_id }
+  let(:topic_admin) { pubsub.topic_admin }
+  let(:aws_role_arn) { "arn:aws:iam::111111111111:role/fake-role-name" }
+  let(:gcp_service_account) { "fake-service-account@project.iam.gserviceaccount.com" }
+  let(:subscription_admin) { pubsub.subscription_admin }
+
+  before do
+    pubsub_emulator_host = ENV["PUBSUB_EMULATOR_HOST"]
+    if pubsub_emulator_host.nil?
+      raise "PUBSUB_EMULATOR_HOST env variable must be set. Please follow instructions at https://cloud.google.com/pubsub/docs/emulator"
+    end
+  end
+
+  after do
+    @topic = topic_admin.get_topic topic: pubsub.topic_path(topic_id)
```

Comment:
You don't need the `@` if you're just using `topic` as a local variable in this block. The `@` sigil signals an instance variable, which in the case of a test means that it is accessible through the entire test (`before`, `it`, and `after` blocks) since they all execute in the context of the same test object.